server/src/db/Seeder.js

```diff
@@ -24,6 +24,33 @@ class Seeder {
       restaurant: "Mochiko Hawaiian",
       description: "To die for."
     });
+
+    await User.query().insert({
+      email: "testUser@email.com",
+      cryptedPassword: "password"
+    })
+    
```

Comment:
I would just ask we make the spacing in this file consistent. Either have an empty line between each query or no empty line